rules.mk

```diff
@@ -278,12 +278,20 @@ PKG_CONFIG:=$(STAGING_DIR_HOST)/bin/pkg-config
 
 export PKG_CONFIG
 
+HOST_FLAGS_OPT:=$(if $(CONFIG_OPTIMIZE_HOST_TOOLS),$(call qstrip,$(CONFIG_HOST_FLAGS_OPT)))
+HOST_FLAGS_OPT:=$(if $(HOST_FLAGS_OPT),$(HOST_FLAGS_OPT),-O2)
```

Comment:
Pack this under a single condition.

Reply:
It's not possible due to promise "`If not set or empty, defaults to "-O2".`"

Reply:
`HOST_FLAGS_OPT:=$(if $(CONFIG_OPTIMIZE_HOST_TOOLS),$(call qstrip,$(CONFIG_HOST_FLAGS_OPT)),-O2)` ?

Reply:
Seems legit.